README.md

```diff
@@ -1,2 +1,43 @@
-# hw01
-Repository for homework 01
+# Homework 1: Hello, hello!
+
+My name is Cleo Nevakivi-Callanan. I am a MAPSS student focused on history— so I find computers very confusing! I was born in Helsinki but hail from Brooklyn. So yeah!
+
+### Academics and all that
+Currently, I'm researching the experiences of American and Finnish adolescents during World War II for my thesis and working as a research assistant for Prof. Kathleen Belew.
+
+### Interests
+In my free time, I enjoy reading (non-academic books), visiting museums, playing violin, drawing, and doing yoga.
+
+Also... **I LOVE BUNNIES!**
+
+Here is a bunny, that I love:
+
+![Ready for Coachella](https://i2.wp.com/mainetoday.com/wp-content/uploads/2018/10/554956-Bunny-Cuddling.jpg?fit=800%2C533)
+
+Also this very handsome boi:
+
+![Ready for Spring](https://mymodernmet.com/wp/wp-content/uploads/archive/TS39l1veeVBLqUnbvRYj_mumitan2.jpg)
+
+*TL;DR I love bunnies*
+
+[Here are some fun facts about bunnies!](http://mentalfloss.com/article/557234/facts-about-rabbits)
+
+### Computer Science and all that
```

Comment:
Nice list! I think you did a good job explaining where you struggled and how you powered through. You mentioned not knowing much about computers, but it already seems like you've already learned a lot!